dev/merge_arrow_pr.py

```diff
@@ -307,42 +322,47 @@ def resolve_jira(title, merge_branches, comment):
 pr_repo_desc = "%s/%s" % (user_login, base_ref)
 
 if pr["merged"] is True:
-    print "Pull request %s has already been merged, assuming you want to backport" % pr_num
+    print("Pull request %s has already been merged, "
+          "assuming you want to backport" % pr_num)
     merge_commit_desc = run_cmd([
         'git', 'log', '--merges', '--first-parent',
         '--grep=pull request #%s' % pr_num, '--oneline']).split("\n")[0]
     if merge_commit_desc == "":
-        fail("Couldn't find any merge commit for #%s, you may need to update HEAD." % pr_num)
+        fail("Couldn't find any merge commit for #%s, "
+             "you may need to update HEAD." % pr_num)
 
     merge_hash = merge_commit_desc[:7]
     message = merge_commit_desc[8:]
 
-    print "Found: %s" % message
-    maybe_cherry_pick(pr_num, merge_hash, latest_branch)
+    print("Found: %s" % message)
+    # maybe_cherry_pick(pr_num, merge_hash, latest_branch)
```

Comment:
Why is this commented out here?

Also I'm bit confused where this function is defined.